src/test/app/Flow_test.cpp

```diff
@@ -1375,9 +1386,12 @@ struct Flow_test : public beast::unit_test::suite
     {
         using namespace jtx;
         FeatureBitset const ownerPaysFee{featureOwnerPaysFee};
+        FeatureBitset const reducedOffersV2(fixReducedOffersV2);
 
         testLineQuality(features);
         testFalseDry(features);
+        testDirectStep(features - reducedOffersV2);
```

Comment:
Why need to run this test with `reducedOffersV2` enabled? There is no change to `testDirectStep` with/without the amendment.

Reply:
I think it should be removed if it doesn't accomplish anything. It just increases the tests running time without any benefit.